2025-03-09 20:53:43,998 INFO    Thread-237 (_run_job):17292 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205343-yqayxbxf\logs\debug.log
2025-03-09 20:53:43,998 INFO    Thread-237 (_run_job):17292 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205343-yqayxbxf\logs\debug-internal.log
2025-03-09 20:53:43,998 INFO    Thread-237 (_run_job):17292 [wandb_init.py:init():761] calling init triggers
2025-03-09 20:53:43,998 INFO    Thread-237 (_run_job):17292 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'sigmoid', 'batchSize': 64, 'hiddenLayer_Size': 32, 'learningRate': 0.001, 'num_epochs': 5, 'num_hiddenLayers': 5, 'optimizer': 'rmsprop', 'weightDecay': 0.0005, 'weightInit': 'random'}
config: {'num_epochs': 10, 'num_hiddenLayers': 3, 'hiddenLayer_Size': 32, 'weightDecay': 0, 'learningRate': 0.001, 'optimizer': 'Nesterov_Accelerated_GD', 'batchSize': 32, 'weightInit': 'xavier', 'activationFunc': 'sigmoid', '_wandb': {}}
2025-03-09 20:53:44,009 INFO    Thread-237 (_run_job):17292 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-09 20:53:44,011 INFO    Thread-237 (_run_job):17292 [wandb_run.py:_finish():2112] finishing run da6401-assignments/assignment1/yqayxbxf
2025-03-09 20:53:44,012 ERROR   Thread-237 (_run_job):17292 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-09 20:53:44,012 INFO    Thread-237 (_run_job):17292 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 20:53:44,012 INFO    Thread-237 (_run_job):17292 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-09 20:53:44,012 INFO    Thread-237 (_run_job):17292 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 1
2025-03-09 20:53:44,012 INFO    Thread-237 (_run_job):17292 [wandb_run.py:_restore():2322] restore
2025-03-09 20:53:44,012 INFO    Thread-237 (_run_job):17292 [wandb_run.py:_restore():2328] restore done
2025-03-09 20:53:46,363 INFO    Thread-237 (_run_job):17292 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-09 20:53:46,363 INFO    Thread-237 (_run_job):17292 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-09 20:53:46,363 INFO    Thread-237 (_run_job):17292 [wandb_run.py:_footer_sync_info():3917] logging synced files
2025-03-09 20:53:46,367 ERROR   MainThread:17292 [pyagent.py:_run_jobs_from_queue():234] Run yqayxbxf errored:
Traceback (most recent call last):
  File "c:\Python312\Lib\site-packages\wandb\agents\pyagent.py", line 306, in _run_job
    self._function()
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_17292\2599373716.py", line 34, in sweep_hyperParameters
    model = FeedforwardNeuralNetwork(x_train.shape[0], hiddenLayer_size, y_train.shape[0])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_17292\3812825710.py", line 21, in __init__
    for layer_size in hidden_layers:
                      ^^^^^^^^^^^^^
TypeError: 'int' object is not iterable

2025-03-09 20:53:50,387 INFO    MsgRouterThr:17292 [mailbox.py:close():129] Closing mailbox, abandoning 1 handles.
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_setup.py:_flush():67] Configure stats pid to 17292
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_setup.py:_flush():67] Loading settings from C:\Users\prajv\.config\wandb\settings
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_setup.py:_flush():67] Loading settings from c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\settings
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205351-laixy4ej\logs\debug.log
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205351-laixy4ej\logs\debug-internal.log
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000017F6EDD4170>
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():761] calling init triggers
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'tanh', 'batchSize': 64, 'hiddenLayer_Size': 32, 'learningRate': 0.0001, 'num_epochs': 5, 'num_hiddenLayers': 3, 'optimizer': 'momentum', 'weightDecay': 0.5, 'weightInit': 'random'}
config: {'_wandb': {}}
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():784] starting backend
2025-03-09 20:53:51,380 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():788] sending inform_init request
2025-03-09 20:53:51,386 INFO    Thread-242 (_run_job):17292 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-03-09 20:53:51,386 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():798] backend started and connected
2025-03-09 20:53:51,387 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_config_callback():1261] config_cb None None {'activationFunc': 'tanh', 'batchSize': 64, 'hiddenLayer_Size': 32, 'learningRate': 0.0001, 'num_epochs': 5, 'num_hiddenLayers': 3, 'optimizer': 'momentum', 'weightDecay': 0.5, 'weightInit': 'random'}
2025-03-09 20:53:51,389 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-09 20:53:51,415 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():891] updated telemetry
2025-03-09 20:53:51,491 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-09 20:53:53,123 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():990] starting run threads in backend
2025-03-09 20:53:53,399 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_console_start():2375] atexit reg
2025-03-09 20:53:53,399 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-09 20:53:53,399 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-09 20:53:53,400 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-09 20:53:53,401 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-09 20:53:53,405 INFO    Thread-242 (_run_job):17292 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205353-laixy4ej\logs\debug.log
2025-03-09 20:53:53,406 INFO    Thread-242 (_run_job):17292 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205353-laixy4ej\logs\debug-internal.log
2025-03-09 20:53:53,406 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():761] calling init triggers
2025-03-09 20:53:53,406 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'tanh', 'batchSize': 64, 'hiddenLayer_Size': 32, 'learningRate': 0.0001, 'num_epochs': 5, 'num_hiddenLayers': 3, 'optimizer': 'momentum', 'weightDecay': 0.5, 'weightInit': 'random'}
config: {'num_epochs': 10, 'num_hiddenLayers': 3, 'hiddenLayer_Size': 32, 'weightDecay': 0, 'learningRate': 0.001, 'optimizer': 'Nesterov_Accelerated_GD', 'batchSize': 32, 'weightInit': 'xavier', 'activationFunc': 'sigmoid', '_wandb': {}}
2025-03-09 20:53:53,406 INFO    Thread-242 (_run_job):17292 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-09 20:53:53,411 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_finish():2112] finishing run da6401-assignments/assignment1/laixy4ej
2025-03-09 20:53:53,411 ERROR   Thread-242 (_run_job):17292 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-09 20:53:53,411 INFO    Thread-242 (_run_job):17292 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 20:53:53,411 INFO    Thread-242 (_run_job):17292 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-09 20:53:53,411 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 1
2025-03-09 20:53:53,411 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_restore():2322] restore
2025-03-09 20:53:53,411 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_restore():2328] restore done
2025-03-09 20:53:55,805 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-09 20:53:55,805 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-09 20:53:55,806 INFO    Thread-242 (_run_job):17292 [wandb_run.py:_footer_sync_info():3917] logging synced files
2025-03-09 20:53:55,813 ERROR   MainThread:17292 [pyagent.py:_run_jobs_from_queue():234] Run laixy4ej errored:
Traceback (most recent call last):
  File "c:\Python312\Lib\site-packages\wandb\agents\pyagent.py", line 306, in _run_job
    self._function()
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_17292\2599373716.py", line 34, in sweep_hyperParameters
    model = FeedforwardNeuralNetwork(x_train.shape[0], hiddenLayer_size, y_train.shape[0])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_17292\3812825710.py", line 21, in __init__
    for layer_size in hidden_layers:
                      ^^^^^^^^^^^^^
TypeError: 'int' object is not iterable

2025-03-09 20:54:05,885 INFO    MsgRouterThr:17292 [mailbox.py:close():129] Closing mailbox, abandoning 1 handles.
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_setup.py:_flush():67] Configure stats pid to 17292
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_setup.py:_flush():67] Loading settings from C:\Users\prajv\.config\wandb\settings
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_setup.py:_flush():67] Loading settings from c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\settings
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205406-226g08xu\logs\debug.log
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205406-226g08xu\logs\debug-internal.log
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000017F6EDDE720>
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():761] calling init triggers
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'sigmoid', 'batchSize': 16, 'hiddenLayer_Size': 32, 'learningRate': 0.0001, 'num_epochs': 5, 'num_hiddenLayers': 5, 'optimizer': 'adam', 'weightDecay': 0.5, 'weightInit': 'xavier'}
config: {'_wandb': {}}
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():784] starting backend
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():788] sending inform_init request
2025-03-09 20:54:06,809 INFO    Thread-247 (_run_job):17292 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-03-09 20:54:06,824 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():798] backend started and connected
2025-03-09 20:54:06,825 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_config_callback():1261] config_cb None None {'activationFunc': 'sigmoid', 'batchSize': 16, 'hiddenLayer_Size': 32, 'learningRate': 0.0001, 'num_epochs': 5, 'num_hiddenLayers': 5, 'optimizer': 'adam', 'weightDecay': 0.5, 'weightInit': 'xavier'}
2025-03-09 20:54:06,827 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-09 20:54:06,843 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():891] updated telemetry
2025-03-09 20:54:06,885 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-09 20:54:08,687 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():990] starting run threads in backend
2025-03-09 20:54:08,929 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_console_start():2375] atexit reg
2025-03-09 20:54:08,930 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-09 20:54:08,931 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-09 20:54:08,931 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-09 20:54:08,932 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-09 20:54:08,936 INFO    Thread-247 (_run_job):17292 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205408-226g08xu\logs\debug.log
2025-03-09 20:54:08,936 INFO    Thread-247 (_run_job):17292 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_205408-226g08xu\logs\debug-internal.log
2025-03-09 20:54:08,936 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():761] calling init triggers
2025-03-09 20:54:08,936 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'sigmoid', 'batchSize': 16, 'hiddenLayer_Size': 32, 'learningRate': 0.0001, 'num_epochs': 5, 'num_hiddenLayers': 5, 'optimizer': 'adam', 'weightDecay': 0.5, 'weightInit': 'xavier'}
config: {'num_epochs': 10, 'num_hiddenLayers': 3, 'hiddenLayer_Size': 32, 'weightDecay': 0, 'learningRate': 0.001, 'optimizer': 'Nesterov_Accelerated_GD', 'batchSize': 32, 'weightInit': 'xavier', 'activationFunc': 'sigmoid', '_wandb': {}}
2025-03-09 20:54:08,936 INFO    Thread-247 (_run_job):17292 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-09 20:54:08,938 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_finish():2112] finishing run da6401-assignments/assignment1/226g08xu
2025-03-09 20:54:08,939 ERROR   Thread-247 (_run_job):17292 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-09 20:54:08,939 INFO    Thread-247 (_run_job):17292 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 20:54:08,939 INFO    Thread-247 (_run_job):17292 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-09 20:54:08,940 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 1
2025-03-09 20:54:08,940 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_restore():2322] restore
2025-03-09 20:54:08,940 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_restore():2328] restore done
2025-03-09 20:54:10,972 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-09 20:54:10,972 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-09 20:54:10,972 INFO    Thread-247 (_run_job):17292 [wandb_run.py:_footer_sync_info():3917] logging synced files
2025-03-09 20:54:10,977 ERROR   MainThread:17292 [pyagent.py:_run_jobs_from_queue():234] Run 226g08xu errored:
Traceback (most recent call last):
  File "c:\Python312\Lib\site-packages\wandb\agents\pyagent.py", line 306, in _run_job
    self._function()
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_17292\2599373716.py", line 34, in sweep_hyperParameters
    model = FeedforwardNeuralNetwork(x_train.shape[0], hiddenLayer_size, y_train.shape[0])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_17292\3812825710.py", line 21, in __init__
    for layer_size in hidden_layers:
                      ^^^^^^^^^^^^^
TypeError: 'int' object is not iterable

2025-03-09 20:54:10,977 ERROR   MainThread:17292 [pyagent.py:_run_jobs_from_queue():245] Detected 3 failed runs in the first 60 seconds, killing sweep.
